### PR TITLE
created model manager

### PR DESCRIPTION
**Pull request by Timur5050**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@ venv/
 .pytest_cache/
 **__pycache__/
 **db.sqlite3
+app/cinema
```
Comment (pavlopro) [inline .gitignore:10]: Add blank line

File: app/managers.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            f"SELECT * FROM {self.table_name}"
+        )
+        return [
+            Actor(*row) for row in all_actors
+        ]
+
+    def update(
+            self,
+            id_to_update: int,
+            first_name: str = "",
```
Comment (pavlopro) [inline app/managers.py:30]: Please remove default "" value for first and last names

File: app/main.py [hunk 3]
```diff
@@ -1,13 +1,5 @@
-# from models import Actor
```
Comment (pavlopro) [inline app/main.py:1]: Leave this code uncommented

**Review by pavlopro — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 3 above)
Comment (pavlopro) [reply app/main.py:1]: Uncomment this code

**Review by pavlopro — CHANGES_REQUESTED** [review]

**Comment by Timur5050** [comment]
it is uncommented, check main.py

File: app/main.py [hunk 4]
```diff
@@ -1,13 +1,5 @@
...
-# from managers import ActorManager
-#
-# if __name__ == "__main__":
-#     Actor.objects = ActorManager()
-#
-#     Actor.objects.create(first_name="Emma", last_name="Watson")
```
Comment (pavlopro) [inline app/main.py:7]: How about this code?

```
#     Actor.objects.create(first_name="Emma", last_name="Watson")
#     Actor.objects.create(first_name="Daniel", last_name="Radclife")
#     print(Actor.objects.all())
#     Actor.objects.update(2, "Daniel", "Radcliffe")
#     print(Actor.objects.all())
#     Actor.objects.delete(1)
#     print(Actor.objects.all())
```

**Review by pavlopro — CHANGES_REQUESTED** [review]

**Comment by Timur5050** [comment]
i thought that it was only for me, for testingm sorry

**Review by Polyakiv-Andrey — APPROVED** [review]
Good job!

**Pull request closed without merging**